itest/lnd_channel_backup_test.go

```diff
@@ -1112,22 +1112,20 @@ func testDataLossProtection(ht *lntest.HarnessTest) {
 	// directly from the miner.
 	ht.FundCoins(btcutil.SatoshiPerBitcoin, carol)
 
-	// timeTravel is a method that will make Carol open a channel to the
-	// passed node, settle a series of payments, then reset the node back
-	// to the state before the payments happened. When this method returns
-	// the node will be unaware of the new state updates. The returned
-	// function can be used to restart the node in this state.
-	timeTravel := func(node *node.HarnessNode) (func() error,
-		*lnrpc.ChannelPoint, int64) {
-
+	// timeTravelDave is a method that will make Carol open a channel to
+	// Dave, settle a series of payments, then Dave back to the state
+	// before the payments happened. When this method returns Dave will
+	// be unaware of the new state updates. The returned function can be
+	// used to restart Dave in this state.
+	timeTravelDave := func() (func() error, *lnrpc.ChannelPoint, int64) {
```

Comment:
I think we passed in the node here because the reference changed when shutting down the node and restarting it. But that doesn't seem to be the case with the new framework anymore, right?

Reply:
hmm didn't realize there was such an issue back then. But yeah the pointer stays unchanged as start and stop methods are built inside the node, so we'd be good here.